DetectorDescription/DetSensitive/tests/scripts/cmpTrackerSdDetailed.py

```diff
@@ -0,0 +1,52 @@
+bitfieldsize = 6
+cellNo = 51
+signed = True
+
+def retrieve(no, mask, offset):
+    id = (no & mask)
+    id = id >> offset
+    if(signed):
+        id = id - 2**bitfieldsize if id >= 2**(bitfieldsize-1) else id
+    return id
+
+def z(cellId):
+    return retrieve(cellId, 0b000000000000111111,0)
+
+def y(cellId):
+    return retrieve(cellId, 0b000000111111000000,1*bitfieldsize)
+
+def x(cellId):
+    return retrieve(cellId, 0b111111000000000000,2*bitfieldsize)
+
+def cell(cellId):
+    return array('d',(x(cellId),y(cellId),z(cellId)))
+
+
+from ROOT import gSystem, THnSparseD
+from EventStore import EventStore
+from array import array
+
```

Comment:
I always prefer `if __name__ == "__main__":` etc if a script as well contains definitions of functions and methods.